code/script.api.php

```diff
@@ -566,11 +566,20 @@ private function _installStraper($parent)
 
 			if (JFile::exists($target . DS . 'version.txt'))
 			{
-				$rawData = JFile::read($target . DS . 'version.txt');
-				$info = explode("\n", $rawData);
-				$straperVersion['installed'] = array(
-					'version' => trim($info[0]), 'date' => new JDate(trim($info[1]))
-				);
+				$rawData = file_get_contents($target . DS . 'version.txt');
```

Comment:
@manojLondhe can you take a look at this. Is strapper needed at all ?

If it's needed please simplify the code @amit4106udale